doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -46,13 +46,20 @@ kieserver.autoScanDeployments=true
 |`kieserver.autoScanDeployments`
 |`true, false`
 |When set to `true`, {KIE_SERVER} scans and deploys all KJAR files available in the Spring Boot application, whether they are declared programmatically or through the Maven plug-in.
-
 |===
-
 +
 When both of these properties are set to `true`, the Spring Boot {KIE_SERVER} automatically scans for deployable artifacts that are packaged within the Spring Boot application using the `package-dependencies-kjar` Maven goal of the `kie-maven-plugin` plug-in.
 +
-. In the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service` directory, add the following Maven plug-in in the Spring Boot `service/pom.xml` file where `<GROUP_ID>`, `<ARTIFACT_ID>`, and `<VERSION>` are the group, artifact, and version (GAV) of a KJAR artifact that your project uses. You can find these values in the `pom.xml` file that is located in the KJAR source directory.
+If your business application will run in an {OPENSHIFT} pod or in any other environment where the current directory is not writable, add the `spring.jta.log-dir` parameter that points to a writable location, for example:
```

Comment:
"parameter that points to a writable location, for example:" Using , for example, blah blah blah is fine, but for end of sentence use:

parameter that points to a writable location. For example: